fix-code-1/main.cpp

```diff
@@ -6,11 +6,11 @@ using namespace std;
 int main() {
     int DAYS_PER_YEAR = 365;
```

Comment:
this should not be capitalized. variables should be lower snake case

Reply:
Done.